lib/jmespath/nodes/function.rb

```diff
@@ -323,10 +333,12 @@ class ValuesFunction < Function
       def call(args)
         if args.count == 1
           value = args.first
-          if hash_like?(value)
-            value.values
-          elsif Array === value
-            value
+          if value.respond_to?(:to_hash)
+            value.to_hash.values
+          elsif value.is_a?(Struct)
+            value.to_h.values
```

Comment:
Can just call `values` on the struct instead of calling `to_h` first.

Reply:
thanks